compiler/src/iree/compiler/Dialect/Flow/Transforms/FormScalarDispatches.cpp

```diff
@@ -61,7 +61,8 @@ static bool isComputeOperation(Operation *op) {
     return true;
   }
   if (op->getDialect() == context->getLoadedDialect<tensor::TensorDialect>()) {
-    return !isa<tensor::CastOp, tensor::EmptyOp>(op);
+    return !isa<tensor::CastOp, tensor::CollapseShapeOp, tensor::EmptyOp,
+                tensor::ExpandShapeOp>(op);
   }
   return false;
```

Comment:
So all StableHLO's if it made it here would be marked as non-compute?

Reply:
yeah, at this point everything is expected to be Linalg ops.